atlp-rwanda/e-commerce-bitcrafters-bn
# #187364723 - Setup docker for backend application

**What does this PR do** 
Setting Up docker configurations 

**Task Included**
1. Initializing a Dockerfile
2. Setting up a docker-compose.yaml file that runs two services
    - Express Server
    - Postgress Database
  
**How should this be manually tested?**
- Before cloning the repo make sure that you have installed docker in your local machine
- Make sure that you fill in all necessary environment variables required
- Clone the repo and install the required packages
- cd into the repo in the terminal and run 
       - 'docker-compose up --build' : which will build the images and container
       - 'docker run -p e-commerce-bitcrafters-bn' : which will run the image
- Other necessary commands
    - 'docker login' : which will allow you to log into your docker account (Better if you have a desktop docker app)
    - 'docker compose up': It wakes up the container
    - 'docker compose down': It sleeps the container

**Any background context you want to provide?**
- Install docker
- Add docker extension on your editor for easy configurations

**Pivotal Tracker stories**
#187364723
